recipes/pyfish/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyfish" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2d1d814b0dcdcc5da3522310787ce5e12ec8b9d2600f0f632316cc0e295ca217
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pyfish = pyfish.main:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib-base >=3.0
+    - numpy >=1.14
+    - pandas >=1.0
+    - pip
+    - pytest >=3.0
+    - python
+    - scipy >=1.0
```

Comment:
You only need to list packages necessary for `pip install`. That should be only `python` and `pip`